### ISSUE TITLE
Exercise bodies are rendered instead of exercise comments

### ISSUE THREAD

**Issue by dialelo** [47deg/scala-exercises#187]
![prepended](https://cloud.githubusercontent.com/assets/409039/13006096/c5b19482-d186-11e5-9877-384847b8675c.png)

Instead of rendering the comment body of the method docstring before the exercise body, a plaintext version of the exercise body is rendered instead. For the record, here is the Scala code of such exercise:

```scala
  /**
    * Because `Xor` is right-biased, it is possible to define a `Monad` instance for it. You
    * could also define one for `Either` but due to how it's encoded it may seem strange to fix a
    * bias direction despite it intending to be flexible in that regard. The `Monad` instance for
    * `Xor` is consistent with the behavior of the data type itself, whereas the one for `Either`
    * would only introduce bias when `Either` is used in a generic context (a function abstracted
    * over `M[_] : Monad`).
    *
    * Since we only ever want the computation to continue in the case of `Xor.Right` (as captured
    * by the right-bias nature), we fix the left type parameter and leave the right one free.
    *
    * {{{
    * import cats.Monad
    *
    * implicit def xorMonad[Err]: Monad[Xor[Err, ?]] =
    *   new Monad[Xor[Err, ?]] {
    *     def flatMap[A, B](fa: Xor[Err, A])(f: A => Xor[Err, B]): Xor[Err, B] =
    *       fa.flatMap(f)
    *
    *     def pure[A](x: A): Xor[Err, A] = Xor.right(x)
    *   }
    * }}}
    *
    */
  def xorMonad(res0: Int, res1: String) = {
    import cats.data.Xor

    val right: String Xor Int = Xor.right(5)
    right.flatMap(x => Xor.right(x + 1)) should be(Xor.right(res0))

    val left: String Xor Int = Xor.left("Something went wrong")
    left.flatMap(x => Xor.right(x + 1)) should be(Xor.left(res1))
  }
```

**Comment by andyscott** [comment]
Fixed in #191

**Comment by rafaparadela** [comment]
@andyscott IMHO I think we should solve this issue  as soon as possible in order to be able to generate content properly.
